packages/material-ui-lab/src/TreeItem/TreeItem.test.js

```diff
@@ -864,10 +863,13 @@ describe('<TreeItem />', () => {
 
           fireEvent.keyDown(getByTestId('one'), { key: '*' });
 
+          expect(toggleSpy.args[0][1]).to.have.length(3);
```

Comment:
We need to check the change handler here because eight will never have aria-expanded set due to a check when applying that attribute.